x-pack/plugin/apm-data/src/main/resources/component-templates/logs-apm.app-fallback@ilm.yaml

```diff
@@ -8,4 +8,4 @@ template:
     index:
       lifecycle:
         name: logs-apm.app_logs-default_policy
-        prefer_ilm: false
+        prefer_ilm: true
```

Comment:
Another question I have is about why we are hardcoding `prefer_ilm: true` here, rather than removing it entirely? Removing it would allow this component template to be used in a more composable way, and since the default is "true", it would be functionally the same.

Reply:
I explicitly set `prefer_ilm` to `true` since our recommendation depends on it. My thought process was that even if we decide to change the default internally it would not break us but I don't think that's a valid consideration. Let me make the changes.